web/.nginx/route-guard.js

```diff
@@ -0,0 +1,32 @@
+// Code in this file is in NJS language https://nginx.org/en/docs/njs/
+
+// eslint-disable-next-line
+let rawRegExps = require('fs').readFileSync(
```

Comment:
Why not `import` if we use `export`?

Reply:
NJS can't import JSON this way. But with fresh mind I realized that we can create module instead of json and switched to module in abdc62c9740c78dd05082d7572c5f2eba1a508bb